src/api/departures/schema.ts

```diff
@@ -0,0 +1,28 @@
+import Joi from 'joi';
+
+export const getStopsNearestRequest = {
+  query: Joi.object({
+    latitude: Joi.number().required(),
+    longitude: Joi.number().required(),
+    maximumDistance: Joi.number(),
+    maximumResults: Joi.number(),
+    filterByInUse: Joi.boolean()
```

Comment:
Er `maximumDistance` og `filterByInUse` tiltenkt bruk i mittatb-app?

Reply:
Tenker det kan være greit å tweake på mens jeg utvikler avganger-fanen i appen, men ser ikke for meg det blir mulig å sette for brukeren, hvis det er det du tenker på.

Reply:
Tenkte bare på om feltene kommer til å bli brukt av app, uavhengig om det eksponeres til brukeren eller ikke. Tenker det er greit med 1 til 1 forhold mellom bruk og api-implementasjon. Men det er bare å ha de der under testing ja.

Reply:
Ser den! Har vært litt problemer på Entur sin side også, så må sikkert gjøre litt forandringer når det er ordnet opp i, og jeg faktisk begynner å bruke det.

On second thought, hadde det kanskje vært lurt å heller legge inn parametre som kan brukes til pagination, slik at man kan laste flere enn default antall stoppesteder.